mmv1/products/alloydb/Cluster.yaml

```diff
@@ -488,7 +488,6 @@ properties:
     properties:
       - !ruby/object:Api::Type::String
         name: "primaryClusterName"
-        immutable: true
```

Comment:
Note: To perform secondary cluster promotion, a user would remove the secondaryConfig field. The promotion logic is implemented by modifying the update steps in pre_update. So this field needs to mutable.